lms/djangoapps/verify_student/views.py

```diff
@@ -718,16 +721,13 @@ def submit_photos_for_verification(request):
     # then try to do that before creating the attempt.
     if request.POST.get('full_name'):
         try:
-            profile_api.update_profile(
```

Comment:
@wedaly Here is the change from your update_profile to the newer method. I took care to keep all the behavior the same (to the best of my knowledge...).